Taskfile.yaml

```
@@ -0,0 +1,192 @@
+    cmds:
+      - atlas migrate rollback --env development
+
+  migrate-status:
+    desc: "Check migration status"
+    cmds:
+      - atlas migrate status --env development
+
+  migrate-new:
+    desc: "Generate migration from GORM models"
+    cmds:
+      - atlas migrate diff add_changes --env development 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)